concrete/src/Updater/Migrations/Migrations/Version20170316000000.php

```diff
@@ -0,0 +1,48 @@
+<?php
+namespace Concrete\Core\Updater\Migrations\Migrations;
+
+use Concrete\Core\Express\EntryList;
+use Concrete\Core\Updater\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Class Version20170316000000
+ * @package Concrete\Core\Updater\Migrations\Migrations
+ *
+ * This migration is to find all entity search columns and refresh the database schema due to core changes that allows
+ * search columns to be longer than 255 chars.
+ * After refreshing all columns we go though all entities and re-fill the search indexes so that any values that are
+ * longer than 255 chars will now be indexed properly.
+ */
+class Version20170316000000 extends AbstractMigration
+{
+    public function up(Schema $schema)
+    {
+        // Get all the entities ALL THE ENTITIES
+        $entities = \Core::make('express')->getEntities(true)->findAll();
+        foreach ($entities as $entity) {
+            /* @var $entity \Concrete\Core\Entity\Express\Entity */
+            $category = $entity->getAttributeKeyCategory();
+            // Get all the attributes for the entity
+            foreach ($entity->getAttributes() as $key) {
+                // Refresh the column from the attribute's search index definition
+                $category->getSearchIndexer()->refreshRepositoryColumns($category, $key);
```

Comment:
Since this doesn't exist on the interface, we should have some kind of safe failure for when this isn't implemented, no?